### ISSUE TITLE
Route /v1/hooks/events does not exist in gateway client

### ISSUE THREAD

**Issue by vietdht** [safe-global/safe-infrastructure#169]
I am following this guide to setup safe locally [Running our services locally](https://github.com/safe-global/safe-infrastructure/blob/main/docs/running_locally.md)
At step 5, create webhook and point it to `http://nginx:8000/cgw/v1/hooks/events`, but the problem is that route does not exist in client gateway, cannot find from `http://nginx:8000/cgw/api#` document also

#### Environment Details
.env component version
```
CFG_VERSION=latest
CGW_VERSION=latest
TXS_VERSION=latest
UI_VERSION=latest
EVENTS_VERSION=latest
```

**Comment by moisses89** [comment]
@iamacook is this step still necessary? https://github.com/safe-global/safe-infrastructure/blob/main/docs/running_locally.md#step-5-add-your-webhooks 
As far as I know the CGW is reading the events directly from `Rabbitmq`. Right?

**Comment by PooyaRaki** [comment]
@moisses89 Yes, but we still have a few hooks that can be invoked through the controller. Additionally, we recently introduced a feature that allows all hooks to be triggered via the controller based on your feature flags.

**Comment by PooyaRaki** [comment]
@vietdht The endpoint is excluded from the docs, but you should be able to access it using the `POST` method.
